quinn-proto/src/config.rs

```diff
@@ -836,6 +873,45 @@ impl ServerConfig {
         self.preferred_address_v6 = address;
         self
     }
+
+    /// Maximum number of [`Incoming`][crate::Incoming] to allow to exist at a time before
+    /// rejecting new [`Incoming`][crate::Incoming] automatically.
+    ///
+    /// Should hover around zero unless connection-creating packets are being received faster than
+    /// the application is calling [`Endpoint::accept`][crate::Endpoint::accept] and dealing with
+    /// them.
+    pub fn max_incoming(&mut self, max_incoming: usize) -> &mut Self {
+        self.max_incoming = max_incoming;
+        self
+    }
+
+    /// Maximum number of received bytes to buffer for each [`Incoming`][crate::Incoming] between
+    /// when the [`Incoming`][crate::Incoming] is created and when it is used to
+    /// accept/refuse/retry/ignore the connection attempt (not including the first packet).
+    ///
+    /// Packets received in excess of this limit are dropped. This may cause 0-RTT data to have to
+    /// be retransmitted, and, in some cases, failure of the handshake to complete.
+    pub fn max_buffer_bytes_per_incoming(
```

Comment:
bikeshed for improved consistency with e.g. `crypto_buffer_size`, `datagram_receive_buffer_size`:
```suggestion
    pub fn incoming_buffer_size(
```

Reply:
Done.